edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/SubscriptionProcessor.cs

```diff
@@ -44,7 +44,8 @@ public SubscriptionProcessor(IConnectionManager connectionManager, IInvokeMethod
             this.invokeMethodHandler = Preconditions.CheckNotNull(invokeMethodHandler, nameof(invokeMethodHandler));
             this.pendingSubscriptions = new ConcurrentDictionary<string, ConcurrentQueue<(DeviceSubscription, bool)>>();
             this.processSubscriptionsBlock = new ActionBlock<string>(this.ProcessPendingSubscriptions);
-            deviceConnectivityManager.DeviceConnected += this.DeviceConnected;
+            deviceConnectivityManager.DeviceConnected += this.CloudConnectivityEstablished;
```

Comment:
I'm stumbling on the names here...the event handler that gets called when a device connects to Edge Hub is called `CloudConnectivityEstablished`? I seem to recall that the hub<->cloud connection is entirely separate from the device <->hub connection...

Reply:
Yeah, sorry the names are confusing -
CloudConnectivityEstablished = Edge device connects/reconnects to IoTHub
ClientCloudConnectivityEstablished = EdgeHub connects to IoTHub on behalf of a client

Reply:
Alright. Sounds like the first is a high-level status based on some heuristics. The second is the status of specific, physical connections.